documentation/assemblies/security/assembly-security.adoc

```diff
@@ -10,9 +10,11 @@ Communication between Kafka brokers (interbroker communication), between ZooKeep
 Communication between Kafka clients and Kafka brokers is encrypted according to how the cluster is configured.
 For the Kafka and {ProductName} components, TLS certificates are also used for authentication.
 
-The Cluster Operator automatically sets up TLS certificates to enable encryption and authentication within your cluster.
+The Cluster Operator automatically sets up and renews TLS certificates to enable encryption and authentication within your cluster.
 It also sets up other TLS certificates if you want to enable encryption or TLS authentication between Kafka brokers and clients.
 
+You can install your own Kafka broker certificates on a per-listener basis.
```

Comment:
I wonder if we should call it _your own Kafka listener certificates_ instead. Broker certificates might be confusing since they are not per-broker but per-listener.

Reply:
I like _Kafka listener certificates _, as long as we explain that they are installed for each broker.

Choosing the best terminology is so important.